### PR TITLE
Fix/3575  add tooltips at notification wall buttons and icons

### PR DESCRIPTION
**Pull request by MariePrandstetter**
#3575 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/i18next.scanner/en/translation.json [hunk 1]
```diff
@@ -348,5 +348,22 @@
   "Maximum size: {{size}} MB": "Maximum size: {{size}} MB",
   "Registration date:": "Registration date:",
   "Cover image of {{publicName}}": "Cover image of {{publicName}}",
-  "Change cover": "Change cover"
+  "Change cover": "Change cover", 
+  "Comment_sub": "Comment",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/en/translation.json:352]: maybe "noun" is more clear

```suggestion
  "Comment_noun": "Comment", 
```

File: frontend/i18next.scanner/en/translation.json [hunk 2]
```diff
@@ -348,5 +348,22 @@
...
-  "Change cover": "Change cover"
+  "Change cover": "Change cover", 
+  "Comment_sub": "Comment", 
+  "New space": "New space", 
+  "Space updated": "Space updated",
+  "Space deleted": "Space deleted",
+  "Space restored": "Space restored",
+  "Status updated": "Status updated", 
+  "Content updated": "Content updated", 
+  "Item deleted": "Item deleted",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/en/translation.json:359]: Item and content are the same thing? If yes I would say to use "content" to both.

File: frontend/i18next.scanner/fr/translation.json [hunk 3]
```diff
@@ -348,5 +348,22 @@
...
   "Maximum size: {{size}} MB": "Taille maximale: {{size}} Mo",
   "Cover image of {{publicName}}": "Image de couverture de {{publicName}}",
-  "Change cover": "Changer la couverture"
+  "Change cover": "Changer la couverture", 
+  "Comment_sub": "Commentaire", 
+  "New space": "Nouvel espace", 
+  "Space updated": "Espace mis à jour",
+  "Space deleted": "Espace supprimé",
+  "Space restored": "Espace restauré",
+  "Status updated": "Changement de statut",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json:357]: to have the same wording as `"Space updated": "Espace mis à jour"` maybe 
```suggestion
  "Status updated": "Statut mis à jour", 
```
(if you decide to apply, do the same fir content)

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/i18next.scanner/en/translation.json (hunk 2 above)
Comment (Skylsmoi) [reply frontend/i18next.scanner/en/translation.json:359]: I agree that it could be nice to change every instance of "Item" to "Content" if it fits

Comment (MariePrandstetter) [reply frontend/i18next.scanner/en/translation.json:359]: Yes they are actually, I'll update

File: frontend/i18next.scanner/fr/translation.json [hunk 4]
```diff
@@ -348,5 +348,22 @@
...
   "Cover image of {{publicName}}": "Image de couverture de {{publicName}}",
-  "Change cover": "Changer la couverture"
+  "Change cover": "Changer la couverture", 
+  "Comment_noun": "Commentaire", 
+  "New space": "Nouvel espace", 
+  "Space updated": "Espace mis à jour",
+  "Space deleted": "Espace supprimé",
+  "Space restored": "Espace restauré",
+  "Status updated": "Statut mis à jour", 
+  "Content updated": "Changement de contenu",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json:358]:
```suggestion
  "Content updated": "Contenu mis à jour",
```

File: frontend/i18next.scanner/pt/translation.json [hunk 5]
```diff
@@ -348,5 +348,22 @@
...
-  "Change cover": "Mudar a capa"
+  "Change cover": "Mudar a capa", 
+  "Comment_noun": "Comentário", 
+  "New space": "Novo espaço", 
+  "Space updated": "Espaço actualizado",
+  "Space deleted": "Espaço eliminado",
+  "Space restored": "Espaço restaurado",
+  "Status updated": "Status actualizado", 
+  "Content updated": "Conteúdo actualizado", 
+  "Content deleted": "Elemento eliminado",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:359]:
```suggestion
  "Content deleted": "Conteúdo eliminado", 
```

File: frontend/i18next.scanner/pt/translation.json [hunk 6]
```diff
@@ -348,5 +348,22 @@
...
+  "Change cover": "Mudar a capa", 
+  "Comment_noun": "Comentário", 
+  "New space": "Novo espaço", 
+  "Space updated": "Espaço actualizado",
+  "Space deleted": "Espaço eliminado",
+  "Space restored": "Espaço restaurado",
+  "Status updated": "Status actualizado", 
+  "Content updated": "Conteúdo actualizado", 
+  "Content deleted": "Elemento eliminado", 
+  "Content restored": "Elemento restaurado",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:360]:
```suggestion
  "Content restored": "Conteúdo restaurado", 
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

**Review by gbonaspetti — APPROVED** [review]

**Review by Skylsmoi — APPROVED** [review]

**Review by gbonaspetti — APPROVED** [review]

**Review by raphj — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
I start test on this pr.

**Comment by PhilippeAccorsi** [comment]
Tooltip not working whis mouseover on icon (just working on mouseover on test)
Screenshot_2021-02-19 Tableau de bord · 2 · Tracim

**Comment by PhilippeAccorsi** [comment]
the tooltip indicate "new space" but is the new folder here
Screenshot_2021-02-19 (4) Fil d'activités · Tracim

**Comment by PhilippeAccorsi** [comment]
I don't know if we need to add tooltip for notification disabled by default (visible in development.ini)
~~~
### Notifications ###
# Comma-separated events listed in this setting are not displayed in notifications to the user.
# Possible events are in the form ENTITY_TYPE.EVENT_TYPE or ENTITY_TYPE.EVENT_TYPE.CONTENT_TYPE where:
# ENTITY_TYPE  = user | workspace | content | mention | workspace_member
# EVENT_TYPE   = created | modified | deleted | undeleted
# CONTENT_TYPE = html-document | file | thread | folder | comment
# To include all notifications, use web.notifications.excluded = ""
; web.notifications.excluded = user.created, user.modified, user.deleted, user.undeleted, workspace.modified, workspace.deleted, workspace.undeleted, workspace_member.modified, content.modified
~~~

**Comment by PhilippeAccorsi** [comment]
Tested on branch.

**Review by Skylsmoi — APPROVED** [review]

**Pull request merged**